ios-base/Makefile

```diff
@@ -1,5 +1,6 @@
 init:
 	xcodegen
```

Comment:
[q]
How is XcodeGen expected to install?(Mint or HomeBrew?)

It would be better using Mint, and create Mintfile.

Reply:
I don' t think XcodeGen's install way. 😢 (I installed by HomeBrew)

Why is using Mint better?

Reply:
Mint (especially using Mintfile) can fix the version of XcodeGen like dependency tools(e.g. CocoaPods, Carthage) 
Considering the release frequency of XcodeGen, I think it's version is better to be controlled.

Reply:
I see. I will add Mintfile to control XcodeGen's version. Thanks!

Reply:
I did it 👍 https://github.com/DroidKaigi/conference-app-2020/pull/447/commits/fa63299e638a4a611f47cd94262417f83e81e172